src/gmm_utils.erl

```diff
@@ -141,6 +145,13 @@ split_edge_id(Bin) ->
 %% More complex functions
 %%%---------------------------
 
+-spec list_other_zones() -> list(binary()).
+list_other_zones() ->
+    [<<"zone1">>, <<"zone2">>, <<"zone3">>] -- [gmm_utils:zone_id()].
```

Comment:
Interesujące podejście, ale chyba czytelniejszy było by zastosowanie `lists:filter`

Reply:
Czy ja wiem? Operator -- jest krótszy niż funkcja filter, plus intencja jest jasna. Można zrobić to tak:

```suggestion
    AllZones = [<<"zone1">>, <<"zone2">>, <<"zone3">>],
    AllZones -- [gmm_utils:zone_id()].
```

Reply:
Nie, to raczej bez sensu. Ogólnie jak już, to wolę to co było wcześniej. W sumie może zostać

Reply:
to zostaje jak jest. i tak kiedyś trzeba będzie zrobić czytanie zon ze zmiennych środowiskowych albo jakiegoś configu